### PR TITLE
feat: single annotation download dialog step 1

### PR DESCRIPTION
**Pull request by kne42**
#435

- fixes some copy bugs with file format dropdown
- file format dropdown now auto-initializes to the first value in its list
- query params can be changed while overwriting the previous history
- 'via AWS S3' -> 'via AWS CLI'
- single annotation download dialog query params, state, hooked up

## Demo

https://github.com/chanzuckerberg/cryoet-data-portal/assets/29165011/16c010ed-e7bf-4ac4-b28e-29607642b2e6

### CODE REVIEW

**Comment by kne42** [comment]
live demo: https://dev-kira-1.cryoet.dev.si.czi.technology/

File: frontend/packages/data-portal/app/components/Run/AnnotationTable.tsx [hunk 1]
```diff
@@ -291,9 +291,12 @@ export function AnnotationTable() {
                 sdsType="primary"
                 sdsStyle="minimal"
                 onClick={() =>
-                  openTomogramDownloadModal({
+                  openAnnotationDownloadModal({
                     datasetId: run.dataset.id,
                     runId: run.id,
+                    annotationId: annotation.id,
+                    // FIXME: are we supposed to only access the 1st option? (this is how it is done elsewhere)
```
[the remark this answers is not in the document]
Comment (kne42) [reply frontend/packages/data-portal/app/components/Run/AnnotationTable.tsx:298]: will this change in the future? @manasaV3

**Review by Janeece — COMMENTED** [review]
Thanks @kne42! Here's my feedback (sorry if i called out anything not included in this ticket):

- [ ] Header is missing "step 1 of 2" overline text 
   Screenshot 2024-03-07 at 4 46 28 PM

- [ ] When i look at the file format tooltip for segmentation mask, the styling seems to be broken? 
   Screenshot 2024-03-07 at 4 42 49 PM

- [ ] Is .ndjson not format option for segmentation mask annotations? I dont see it in the demo so I want to confirm if this was a change/bug/incorrect in the design (cc @manasaV3 ?)
   Screenshot 2024-03-07 at 4 53 07 PM

- [ ] The "next" button should be active because the field always has a value (please ignore if this is just because its not hooked up to step 2 dialog)

**Comment by kne42** [comment]
> When i look at the file format tooltip for segmentation mask, the styling seems to be broken?

that's odd. I'm not encountering it on the dev site/locally, could you provide steps to reproduce?

> Is .ndjson not format option for segmentation mask annotations? I dont see it in the demo so I want to confirm if this was a change/bug/incorrect in the design (cc @manasaV3 ?)

.ndjson will only appear in the list of options if the file exists in the backend

> The "next" button should be active because the field always has a value (please ignore if this is just because its not hooked up to step 2 dialog)

yeah it'll be fixed when step 2 is implemented

**Review by Janeece — APPROVED** [review]

**Pull request merged**